charts/self-hosted/charts/aserto-directory/templates/deployment.yaml

```
@@ -64,6 +64,17 @@ spec:
               cpu: "2000m"
           image: ghcr.io/aserto-dev/directory:{{ .Values.global.directory.version }}
           args: ["--config", "/config/config.yaml", "run"]
+          env:
+            - name: DIRECTORY_DB_WRITER_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ tpl (.Values.global.postgresql.existingSecret) . }}"
+                  key: postgres-password
+            - name: DIRECTORY_DB_READER_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ tpl (.Values.global.postgresql.existingSecret) . }}"
+                  key: postgres-password
```

The previous suggestion would require a change here as well

```suggestion
                  key: "{{ .Values.global.postgresql.secretKeys.adminPasswordKey }}"
```